pkg/operator2/oauth.go

```diff
@@ -136,3 +148,173 @@ func (c *osinOperator) handleOAuthConfig(configOverrides []byte) (*corev1.Config
 		},
 	}, nil
 }
+
+func convertProviderConfigToOsinBytes(providerConfig *configv1.IdentityProviderConfig) []byte {
```

Comment:
This will need to be able to `error`.  At the caller, for now, just log the error and keep going to the next IDP.